backend/src/main/java/eu/clarin/switchboard/resources/DataResource.java

```diff
@@ -17,20 +22,18 @@
 import javax.ws.rs.core.*;
 import java.io.*;
 import java.net.URI;
-import java.net.URLConnection;
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
-import java.util.Comparator;
-import java.util.List;
-import java.util.Map;
-import java.util.UUID;
+import java.util.*;
 import java.util.stream.Collectors;
+import java.util.zip.ZipException;
 import java.util.zip.ZipFile;
 
 @Path("/api/storage")
 public class DataResource {
     private static final Logger LOGGER = LoggerFactory.getLogger(DataResource.class);
     private static final int MAX_INLINE_CONTENT = 4 * 1024;
+    private static final long MAX_ZIP_ENTRIES = 4 * 1024;
```

Comment:
Is this just about number of entries displayed? Is there any relation between this and DataStoreConfig.maxFiles ?

Reply:
I renamed it to MAX_OUTLINE_ENTRIES, it's a limit necessary because for more entries the UI becomes slow and unresponsive. It is not the same as maxFiles, that's the total limit of files in storage for all users.

 https://github.com/clarin-eric/switchboard/pull/206/commits/7c28046b0ae55065b09112e005c6ad407800686e#diff-7c9d6e6d1582338bd85c5a4ea194309d11b404b62e8eedf28d7e220d7bb77084R24